scripts/tests/relayed-dns-etc-resolvconf-portal-down.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "./scripts/tests/lib.sh"
+
+HTTPBIN=dns.httpbin
+
+function run_test() {
+    echo "# Access httpbin by DNS"
+    client_curl_resource "$HTTPBIN/get"
+
+    echo "# Make sure it's going through the tunnel"
+    client_nslookup "$HTTPBIN" | grep "100\\.96\\.0\\."
+}
+
+install_iptables_drop_rules
+trap remove_iptables_drop_rules EXIT # Cleanup after us
+
+run_test
+
+docker compose stop api
```

Comment:
If we always call it "portal", can we rename "api" to "portal" in `docker-compose.yml` in another PR? At a glance I thought maybe the filename was mismatched.

Reply:
Well `portal` is `web` and `api` so it would require renaming huge swaths of the CI configuration, infra, and Elixir codebase unless we were ok introducing tech debt.

I think we can just rename these to `api` instead of the other way around?

Reply:
Sure. So when we say "portal" it's actually two different services, an API that everything uses, and the web server that admins use?

Reply:
Yeah pretty much